src/vm/hints/hint_utils.rs

```diff
@@ -129,11 +132,11 @@ pub fn get_mut_list_ref_from_scope<'a>(
 }
 
 pub fn get_list_u64_from_scope_ref<'a>(
-    vm: &'a mut VirtualMachine,
+    exec_scopes: &'a mut ExecutionScopes,
```

Comment:
And even here, since the returned ref is immutable.

Reply:
Included in refactor